src/vm.rs

```diff
@@ -1680,7 +1680,18 @@ impl<'a, T: GuestStagePagingMode> FinalizedVm<'a, T> {
         Ok(UmodeTask::execute_request(
             request,
             Some(shared_data),
-            &UmodeTask::only_std_hypcalls,
+            &|hypc| match hypc {
+                u_mode_api::HypCall::ExtSign {
+                    msg_addr: _,
+                    msg_size: _,
+                    sign_addr: _,
+                    sign_size: _,
+                } => {
+                    println!("Signing!");
+                    Ok(())
+                }
+                _ => Err(ExecError::UnhandledHypCall(hypc)),
```

Comment:
So the signing request can no longer panic since HypCall::Panic would take this branch? Or am I missing something?

Reply:
? An error return by this function will panic U-mode. I think you got this backward. I am not filtering all hypercalls to go through here. The external handler is called only after the basic hypercalls (panic, nextop, etc) checked.

Reply:
Ah, so that is what I had missed!